osu.Game/Screens/OnlinePlay/Multiplayer/Match/Playlist/MultiplayerPlaylist.cs

```diff
@@ -121,28 +110,29 @@ private void updateState()
 
                 firstPopulation = false;
             }
```

Comment:
https://github.com/ppy/osu/pull/32250/files#r1998408051

> Can MultiplayerClient not fire ItemAdded on initial join or something to avoid playing these firstPopulation games here? Because I see that at least 2 components are playing the same boolean flag games and at that point it feels rather icky.

This PR no longer changes/moves these lines around, and I'd like to keep them as is for the time being. I don't particularly like the idea of `ItemAdded` being called on initial join, because those are supposed to 1-1 correspond to server-side events.

Reply:
Believe me or not but in that huge diff I actually could not have spotted that this logic already was in `master` to save my life, I just saw stuff changing around making it appear in more places. I'm ok leaving as is.